commands/minecraft.js

```diff
@@ -0,0 +1,44 @@
+const Augur = require("augurbot"),
+  request = require("request-promise-native"),
```

Comment:
Uses `utils/utils.js` later without requiring it.
```suggestion
  request = require("request-promise-native"),
  u = require("../utils/utils),
```